src/pages/SignIn.jsx

```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom";
+import Header from "../components/Header";
+
+export default function SignIn() {
+  return (
+    <div>
+      <Header title={"Welcome back"} />
+      <label htmlFor="username">Username</label>
+      <input id="username" type="email" />
+      <label htmlFor="password">Password</label>
+      <input id="password" type="password" />
+      <Link to={'/'}><button>Sign in</button></Link>
+    </div>
+  );
```

Comment:
You can only do a POST or PATCH with a user that exists in the database so this might be overkill! You could always give the client a set of users that you know exist. Cross that bridge when you come to it!